src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -343,7 +343,7 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
     let modifiedMeta: IChangeableAnalyticsMetaObject = _.extend({}, meta);
     modifiedMeta['JSUIVersion'] = version.lib + ';' + version.product;
 
-    if (result) {
+    if (!(modifiedMeta['contentIDKey'] && modifiedMeta['contentIDValue']) && result) {
```

Comment:
Fake sake of clarity, you could export the condition `modifiedMeta['contentIDKey'] && modifiedMeta['contentIDValue']` to a variable 
```
const hasDefinedContentID = modifiedMeta['contentIDKey'] && modifiedMeta['contentIDValue'];
if (!(hasDefinedContentID) && result) {
```